editions/tw5.com/tiddlers/concepts/ShadowTiddlers.tid

```diff
@@ -11,7 +11,11 @@ tags: Concepts
 
 ShadowTiddlers are tiddlers that are loaded from within [[Plugins]]. Unlike ordinary tiddlers, they don't appear in most lists.
```

Comment:
I think it should start like this: 

```
ShadowTiddlers are tiddlers that are extracted from [[Plugins]] at the wiki startup. Unlike ...
```

Reply:
Good idea @pmario 
I'm closing this and reposting new.